app/assets/javascripts/angular/controllers/mobile/base-mobile.js

```diff
@@ -0,0 +1,48 @@
+(function() {
+  angular.module('wtsApp').controller('baseMobileCtrl', ['$scope', '$http', function($scope, $http) {
+
+    // Set the default to be no permissions.
+    $scope.myPermissions = [];
+
+    $scope.checkPermissions = function(tab) {
+      var myPermissions = $scope.myPermissions;
+      return myPermissions.indexOf(tab) !== -1;
+    }
+
+    // Tabs
+    $scope.setTab = function(tab) {
+      $scope.focusedTab = tab;
+    }
+
+    var apiCall = function() {
+      $http.get('/api/mobile_basic').then(
+      function successCallback(response) {
+        $('#connection-error').hide();
+
+        var result = response['data']['result'];
+
+        $scope.myCountryId = result['team']['id'];
+        $scope.myCountryName = result['team']['team_name'];
```

Comment:
Currently breaks for Russian Federation.  This is because the team name is Russian Federation but the image is Russia.png